api/tests/functional-tests/backend/core/test_annotation.py

```diff
@@ -89,7 +89,7 @@ def test_create_annotation_already_exists_error(
 
     core.create_groundtruths(db, empty_groundtruths)
     core.create_predictions(db, empty_predictions)
-    with pytest.raises(exceptions.DatumAlreadyExistsError):
+    with pytest.raises(exceptions.DatumsAlreadyExistsError):
```

Comment:
nit: `DatumsAlreadyExistError` is probably the more grammatically-correct verbage (removing the 's' from "Exists")